docs/release-notes/release-notes-0.17.0.md

```diff
@@ -117,6 +117,9 @@ on the old value of 10000](https://github.com/lightningnetwork/lnd/pull/7780).
   message now supports all fields that are present in the `OpenChannel` message,
   except for the `funding_shim` and `fundmax` fields.
 
+* The [WalletBalanceRequest](https://github.com/lightningnetwork/lnd/pull/7820)
```

Comment:
nit: Reads a bit cryptic to non-developers who don't know what the `WalletBalanceRequest` refers to. How about "The `WalletBalance` RPC (`lncli walletbalance`) now supports showing the balance for a specific account."?